dev-python/pipenv/pipenv-2020.6.2.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
```

Comment:
```
 * Messages for package dev-python/pipenv-2020.6.2:

 * DISTUTILS_USE_SETUPTOOLS value is probably incorrect
 *   value:    DISTUTILS_USE_SETUPTOOLS=bdepend (default?)
 *   expected: DISTUTILS_USE_SETUPTOOLS=rdepend
```

Reply:
Yes, that's correct. Now that understood the man page this should be `rdepend` if the software uses entry points (and it does).